src/llm/apis/openai_completions.cpp

```diff
@@ -93,10 +93,9 @@ absl::Status OpenAIChatCompletionsHandler::parseCompletionsPart() {
 
 ov::Tensor load_image_stbi(const std::string& imageBytes) {
     int x = 0, y = 0, channelsInFile = 0;
-    constexpr int desiredChannels = 3;
     unsigned char* image = stbi_load_from_memory(
         (const unsigned char*)imageBytes.data(), imageBytes.size(),
-        &x, &y, &channelsInFile, desiredChannels);
+        &x, &y, &channelsInFile, 0);
```

Comment:
What issue is it fixing? Is it handling monochrome images? Shall we have some tests included?